2019-07-24.txt

```diff
@@ -1,326 +1,3 @@
-
-Es wird gebeten, das QSM Referat zu fragen, in welcher Höhe die QSM in den nächsten Jahren weiter existieren werden. Wenn das QSM Referat das nicht weiß, soll es das Rektorat fragen.
-${mathi} fragt nach.
-
-====================
-TOP 5: Stura-Bericht
-====================
-
-Es wird aus dem Stura berichtet. Siehe Protokoll vom Stura
-
-Besonderes:
-
-- am 5.02.2019 kommt der Rektor in den Stura
-- Neuer runder Tisch zu Sperrzeiten in der Altstat wird ins Leben gerufen (Studis sind aus diesen Diskussionen ausgeschlossen)
-- QSM Referat lässt erinnern, dass die im letzten Top genannte Deadline naht.
-- Urabstimmung fürs Semesterticket wurde diskutiert:
-  - eine oder zwei Abstimmungen?: Wochenend und Abendregelung und Semesterticket
-  - Zuerst wird die Urabstimmung abgewartet, dann weiter verhandelt. Daher liegen die Verhandlungen fürs Erste auf Eis.
-  - Stura muss nicht zwingend eine Urabstimmung machen
-- Umsatzsteuern bei Veranstaltungen der VS
-  - Derzeit die Tendenz im Stura: Die Fachschaften, die zu Umastzsteuern führen, zahlen diese Steuern selber.
-  - Bisher gab es kaum Umsatzsteuer in relevanter Höhe. Wurde daher zentral vom Stura bezahlt.
-- Heidelberger Club für Wirtschaft und Kultur will für ein Symposium 5500€ haben. Gab ne ewige Diskussion.
-- Anglistik hat eine Mitgliedschaft im Jugendherrbergsverband beschlossen ohne den Vorsitz zu fragen. Stura ist da schon Mitglied. Die 40€ müssen Menschen in der Anglistik privat bezahlen.
-- Bewirtungsrichtlinie: Interne Veranstaltungen dürfen nur mit 5% des Haushalts und biszu 500€ gefördert werden.
-  - FSWE fällt nicht darunter, Begründungen müssen aber gebracht werden.
-- In 2 Wochen ist Stura Shutdown (Kein Vorsitz und kein Finanzreferat)
-
-Henrik tritt in 3 Sitzungen aus dem Stura aus. Es wird nach NachfolgerInnen in der Mathematik gesucht.
-
-==================================
-TOP 6: Neues aus Studium und Lehre
-==================================
-
-Es wird aus der Sitzung über Geld für Tutoren berichtet.
-Es wird aus Berufungskomissionen berichtet.
-
-====================
-TOP 7: Verschiedenes
-====================
-
-Wirtschaftsplan für 2019. Ist noch nicht fertig, nächste Woche liegt er vor. Falls es Einwände und Ideen für den neuen Wirtschaftsplan gibt, möge man sich bei Jan melden.
-
-Garderobe Medifasching:
-- es wird erklärt was der Medifasching ist
-- Frage: machen / nicht machen
-- Max würde im Notfall die Organisation übernehmen, nächstes Jahr aber definitiv nicht mehr.
-- Geldfrage muss genau fixiert werden.
-- Fragen ob es eine vegetarische Alternative fürs Frühstück gibt
-- Wer helfen möchte, sende eine Mail an max@mathphys.stura.uni-heidelberg.de
-- 01.02.2019 (Freitag) ist der Termin. Ab 15 Uhr Aufbau.
-${chrisb}
-
-Der AK Fest tagt am Montag, den 14.01.2019 ab 18:15 vmtl. in Seminarraum C
-${chrisb} ist eingeladen.
-
-Ende: 19:16
-=======
-:Protocoldude: Mails versandt @ 20:00 27.07.2019
-
-
 Protokoll der gemeinsamen Sitzung vom 24.07.2019
 
 Sitzungsleitung:
```

Comment:
What is this change about?

Reply:
Just deleted a protocol for code validation